execution_workflows/Roar/workflow/config/samples.csv

```diff
@@ -0,0 +1,3 @@
+sample,sample_type
```

Comment:
This file seems to be duplicated (you mention to delete test data, just want to make sure this is also tagged).